ui/dashboards/src/context/DashboardProvider/panel-group-slice.ts

```diff
@@ -171,3 +118,43 @@ export function createPanelGroupSlice(
     },
   });
 }
+
+function convertLayoutsToPanelGroups(
```

Comment:
If you end up being able to remove the `setPanels` and `setPanelGroups` methods from the public API, you might just need to export this for use in `reset`.